### PR TITLE
added sections + changed more details to the end

### PR DESCRIPTION
**Pull request by SimaoBolota-MetaCell**
What changes with this PR
---

- Divided checklist into sections
- changed Error section to the end of the checklist
- More Details section displayed after the checklist

image

### CODE REVIEW

File: napari_hub_cli/checklist/metadata.py [hunk 1]
```diff
@@ -236,12 +241,14 @@ def display_checklist(analysis_result):
...
 
-    # Display additional informations
-    for feature in analysis_result.additionals:
-        console.print(f"  {feature.meta.name}: {feature.result}")
-
     # Display summary result
+    previous_title = None
     for feature in analysis_result.features:
+        if feature.meta.section:
+            if feature.meta.section.title != previous_title:
```
Comment (aranega) [inline napari_hub_cli/checklist/metadata.py:248]: @SimaoBolota-MetaCell, can you compact those two lines in only one `if` with and `and`?

File: napari_hub_cli/checklist/metadata.py [hunk 2]
```diff
@@ -289,3 +296,13 @@ def display_checklist(analysis_result):
             style="red",
         )
         console.print(f"  Recommended file location - {feature.meta.advise_location}")
+    
+    # Display additional informations
+    for feature in analysis_result.additionals:
+        if feature.meta.section:
+            if feature.meta.section.title != previous_title:
```
Comment (aranega) [inline napari_hub_cli/checklist/metadata.py:303]: @SimaoBolota-MetaCell Same as in the code before, if you can compact them in a single `if` using `and` that would be perfect :)

**Review by aranega — CHANGES_REQUESTED** [review]
@SimaoBolota-MetaCell All looks good :) , I would just make two minor modification to avoid the succession of `ifs` in the code

**Comment by SimaoBolota-MetaCell** [comment]
> @SimaoBolota-MetaCell All looks good :) , I would just make two minor modification to avoid the succession of `ifs` in the code

Thank you! All done
Thanks again for the help

**Review by aranega — APPROVED** [review]

**Pull request merged**